gertprofile.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <header>
+        <h1>
+            Gert Cauwenberghs
+        </h1>
+    </header>
+    <hr/>
+    <main>
+        <img src="https://www.thoughtco.com/thmb/WXYMd7DevHhY13xG05Jdw9Q07mA=/1500x0/filters:no_upscale():max_bytes(150000):strip_icc()/tousled-professor-examining-samples-under-microscope-looking-surprised-597071079-58a3ad145f9b58819c90e3b8.jpg" style="width: 300px;">
+        <ul>
+            <li>
+                <b>
```

Comment:
`<b>` is good, but I'd use that for bolding part of some text in a paragraph. `<strong>` makes things a little bit more bold and slightly bigger than `<b>`, so `<strong>` might be a better choice here